cips/cip-031.md

```diff
@@ -5,7 +5,8 @@
 | description    | Dynamically update vesting schedules to include newly earned staking rewards                                            |
 | author         | Dean Eigenmann ([@decanus](https://github.com/decanus)), Marko Baricevic ([@tac0turtle](https://github.com/tac0turtle)) |
 | discussions-to | [Forum Discussion](https://forum.celestia.org/t/cip-lockup-accounts-staking-rewards/1908)                               |
-| status         | Review                                                                                                                   |
+| status         | Last Call                                                                                                               |
+| last-call-date | 2025-04-02                                                                                                              |
```

Comment:
```suggestion
| last-call-deadline | 2025-04-02                                                                                                              |
```

Reply:
I think we can also remove `| requires       | N/A  |` from this page on line 12/13